src/TrackArtist.cpp

```diff
@@ -2182,7 +2182,21 @@ void TrackArtist::DrawClipSpectrum(WaveTrackCache &waveTrackCache,
 
    const SpectrogramSettings::ScaleType scaleType = settings.scaleType;
 
-   const NumberScale numberScale(settings.GetScale(minFreq, maxFreq, rate, true));
+   // nearest frequency to each pixel row from number scale, for selecting
+   // the desired fft bin(s) for display on that row
+   float bins[ hiddenMid.height + 1 ];
```

Comment:
Again, the doubtfully portable dimensioning of an array with a non-constant expression.  alloca?